jbpm-console-ng-business-domain/jbpm-console-ng-business-domain-api/src/main/java/org/jbpm/console/ng/bd/model/TaskDefSummary.java

```diff
@@ -24,14 +24,18 @@
     private long id;
     private String name;
 
-    public TaskDefSummary(long id, String name) {
-        super();
+    public TaskDefSummary(long id, final String name) {
+        this(name);
         this.id = id;
-        this.name = name;
+    }
 
+    public TaskDefSummary(final String name) {
+        super();
+        this.name = name;
     }
 
     public TaskDefSummary() {
+        super();
```

Comment:
interesting as it's always added when you generate constructor via IDE :) I'll leave it for now just to avoid too many updates to PR